docs/quickstart.md

```diff
@@ -174,6 +176,152 @@ sudo tee /etc/containerd/firecracker-runtime.json <<EOF
 EOF
 ```
 
+A similar script for rpm based linux distro e.g. Amazon Linux 2 can be found here
+
+<details>
+
+```
+#!/bin/bash
+
+cd ~
+
+# Install git, Go 1.13, make, curl
+sudo yum -y update
+sudo yum -y install \
+  golang \
+  make \
+  git \
+  curl \
+  e2fsprogs \
+  util-linux \
+  bc \
+  gnupg
+
+
+# Amazon Linux 2 installs go binary is installed in /usr/bin. So no PATH changes required.
+
+cd ~
+
+# Install Docker CE
+# Docker CE includes containerd, but we need a separate containerd binary, built
+# in a later step
+sudo yum -y update
+sudo amazon-linux-extras install -y  docker
+sudo usermod -aG docker $(whoami)
+
+cd ~
+
+# Check out firecracker-containerd and build it.  This includes:
+# * firecracker-containerd runtime, a containerd v2 runtime
+# * firecracker-containerd agent, an inside-VM component
+# * runc, to run containers inside the VM
+# * a Debian-based root filesystem configured as read-only with a read-write
+#   overlay
+# * firecracker-containerd, an alternative containerd binary that includes the
+#   firecracker VM lifecycle plugin and API
+# * tc-redirect-tap and other CNI dependencies that enable VMs to start with
+#   access to networks available on the host
+git clone https://github.com/firecracker-microvm/firecracker-containerd.git
+cd firecracker-containerd
+sudo yum -y install device-mapper
+sg docker -c 'make all image firecracker'
```

Comment:
Does it need sg since the user is in docker group?

Reply:
Even though we add the user to docker group, without running with sg the Gid of a process run by the user is still its primary group. This can be verified by reading /proc/<pid>/status  But if we run with sg the Gid is seen to be of docker group. So i think its better to run with sg.